tests/test_aridanalysis.py

```diff
@@ -1,5 +1,123 @@
 from aridanalysis import __version__
-from aridanalysis import aridanalysis
+from aridanalysis import aridanalysis as aa
+
+import pytest
+
+import pandas as pd
+import numpy as np
+import sklearn
+import statsmodels
+import warnings
+
+import sys, os
+myPath = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, myPath + '/../aridanalysis')
+import error_strings as errors
 
 def test_version():
-    assert __version__ == '0.1.0'
\ No newline at end of file
+    assert __version__ == '0.1.0'
```

Comment:
Don't really know what this is? Might need to adjust based on the current version number

Reply:
Yes, will look up. I don't know if we'd have to manually do this though?